Gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-gem "rails", "6.1.5"
+gem "rails", "7.0.2"
```

Comment:
[Version 7.0.2.3](https://rubygems.org/gems/rails/versions/7.0.2.3) is now available, would it be possible to bump to that version now?